### DISCUSSION TITLE
Heated bed and enclosure

### DISCUSSION THREAD

**Discussion by Blargedy** [jyjblrd/Core-R-Theta-4-Axis-Printer#1] [Ideas]
Do you plan on working on the mechanical design of the printer any more? The features that are missing compared to other printers is a heated bed and enclosed build volume so that you can print with ABS/ASA.

The heated bed is far more important than the enclosure since the enclosure can be jerry rigged together with basically any material, and doesn't need much design work. I'll explore the printer's CAD soon to see if the heated bed can be done with slip rings or something, and talk to fabreeko about fabricating a custom heater pad and machined build plate.

**Comment by Thefonze74065** [comment]
I'd love to see this, but do we know if a heated bed is necessary for non-planar printing?

I assume prints wouldn't warp in the same ways as a traditional printer... 
with some slicer settings, we might even be able to make most layers shrink towards the build plate into the print, rather than upwards away from it; conical layers should shrink in a set direction.

**Reply by Blargedy** [reply]
In order to print ABS/ASA you need an enclosure, kept at 55-80c and a build plate at 110c. The chamber heat is typically applied via the build plate getting and staying hot. You also need to vent the VOC laden air either outside your house, or filter it through a carbon filter in a circulatory fashion, taking the VOCs away with each pass through the carbon filter. Naturally both venting and circulating need an enclosed printer. 

Heated beds are nice to have for PLA but not strictly necessary, as we saw in the video. I print exclusively in ABS/ASA for functional parts, so for my use case both heated bed and enclosure would be needed, while for those printing PLA the enclosure would (maybe) hurt more than it would help due to heat creep and part cooling limitations. Normally enclosures would cause the part cooling to be inadequate due to the hotter air being used, but because this printer takes air from outside, part overheating wouldn't (maybe) be a problem. The heat creep might be fine if you keep the enclosure open or removed, but I'm guessing.

**Reply by Thefonze74065** [reply]
Yes?
I'm mainly referencing PLA - warping happens when plastic cools, and lifts off the bed as it tries to stick to the layers below it, lifting them off the bed.
(Imagine stretching 3 strips of rubber, and layering them - the ends curl up as they shrink back)

Non-planar printing might warp differently, as a 'layer' would shrink differently due to being 3d, versus a 2d layer.

ABS/ASA would 100% require a heated bed and enclosure, but a heated bed might not be as good of an upgrade in terms of PLA, versus on a planar printer.

**Reply by Blargedy** [reply]
Hmm it sounds like the heated bed is optional in your opinion, in which case there is no point in paying extra for the heater. However I would like the option, so my question was if there is any plans to add any features at which point I could recommend implementing milled flat aluminum beds with bed heaters and letting those who don't want those just leave them out. Adding bed heating is the biggest step in making this a modern printer, in my opinion. Other changes will give marginal increasing performance but a heated bed would add functionality that didn't exist in the first place.

**Reply by Thefonze74065** [reply]
I'd also like to see a heated bed on this printer - mainly for ASA parts.

I'm just really curious about how non-planar printing changes existing 3d printer knowledge, such as warping.
If nobody else does first, I'll do a few experiments to test stuff like this.

**Reply by Blargedy** [reply]
If non-planar can help with warping it might help with printing exotic materials that need bed glue and high enclosure temperatures, so testing that is a good idea.

**Reply by TheOnlyMann** [reply]
unrelated to ABS fume issues, how would the heated bed be constantly powered while its rotating on a shaft?
i get slip rings exist, but since like normal printbed draws out ~30 Amps(quick google search, I know nothing about this) ~30A slip rings would probably cost about ~40 bucks(from amazon, your mileage may vary) with possible wear/tear over time
it would be a nice feature to have, but idk if its going to be easy.


https://www.instructables.com/Make-a-Polar-3D-Printer-Spinning-Bed/
Did found some Ender 2 Polar Conversion project that went with the slip ring. Possibly the design can be interchanged with the current one?

**Reply by TheOnlyMann** [reply]
Would it be possible to re-instate Heated bed upgrades from a Delta printer(which beds are already shaped circular, and their magnetic springsteel PEI plates also) and use that alongside a slip ring to make a proper heated bed for the polar printer?
seems like there already exists like these [Anycubic Delta Kossel](https://www.aliexpress.us/item/1005004111369907.html)'s Heated bed upgrade kits.
Due to how everything has to spin you probably need a [4 channel slip ring](https://www.aliexpress.us/item/1005003469962786.html) I guess? Not really knowledgable in any of this field so feel free to correct my mistakes.

**Reply by Blargedy** [reply]
That's not a bad idea to modify a delta printer's bed assembly to cut down on the designing required for something like this. I could see that working.

**Reply by TheOnlyMann** [reply]
Update
looked at some stuff and i think the DC bed is the way to go, for now

DC Bed

 +Pros
 - can use existing products like Siboor Kossel Heated Bed upgrade kit
 - uses DC so less headache

-Cons
 - Requires more Amps thus robust slip ring hardware
 - Requires more robust PSU than a power brick(even more so)

AC Bed

 +Pros
 - Can steal design/specification off from Doron Velta
 - uses less amps so slip rings cost less

-Cons
 - AC so more ways to fry yourself
 - possibly more weight cause silicon heaters tend to be heavy, thus limiting spin

**Reply by Blargedy** [reply]
I also think a DC bed heater would be best, due to the safer voltage and not needing a high voltage SSR or dealing with mains power.

**Comment by jyjblrd** [comment]
I'll tell you guys a secret, the printer already has a heated bed. In the video i mentioned I used a PCB for the buildplate, so I was like I might as well add a PCB trace heatbed at the same time. I spent way too much time writing a script to generate PCB traces for arbitrary shapes so I could add the traces to the bed. It even has zone heating and a ring of RGB LEDs, just for the hell of it lol.
<img width="1131" alt="Screenshot 2025-01-04 at 11 57 03 AM" src="https://github.com/user-attachments/assets/c6515cfa-c692-4662-b1fe-f44ead14a0e1" />



I never bothered to actually connect it up though, but I may at some point. 

cc: @Blargedy @Thefonze74065 @TheOnlyMann

**Reply by jyjblrd** [reply]
The PCB files are in the repo if you want to make it yourself, it's obviously untested

**Reply by TheOnlyMann** [reply]
interesting. I did look up the pcb files but couldn't connect the dots.
that said i think there is still merit to using existing hardware than custom ordering from the shop, so ill still search my endeavors, thanks!

**Reply by jyjblrd** [reply]
Yeah, if I were to go that route I would use a silicon mains voltage heated bed with a slip ring. The amperage would be low enough to use a small slip ring and circular silicon heat beds are common because delta printers use them. But looks like you guys have already compared the pros/cons of that.

**Reply by Blargedy** [reply]
Oh cool. I wonder how much it would cost to get this printed. I hesitate at the untested status but I'll look into it.

**Reply by jyjblrd** [reply]
Prob ~60usd for 5

**Comment by TheOnlyMann** [comment]
at a unrelated note, would it be possible to replace the ball bearings used for a angular contact ball bearings
7008ACBearings for example
